creating_packages/package_information.rst

```diff
@@ -0,0 +1,89 @@
+.. _package_information:
+
+Define the package information
+==============================
+
+When creating a recipe to package a library, it is important to define the information about the package so consumers can get the
+information correctly. Conan achieves this by decoupling the information of the package from the format needed using
+:ref:`generators_reference`, that translate the generic information into the appropriate format file.
+
+This generic information is defined inside the recipe, using the :ref:`method_package_info` method. There you can declare package information
+like the location of the header files, library names, defines, flags...
+
+.. code-block:: python
+
+    from conans import ConanFile
+
+    class MyConan(ConanFile):
+        name = "cool_library"
+        ...
+
+        def package_info(self):
+            self.cpp_info.includedirs = ["include/cool"]
+            self.cpp_info.libs = ["libcool"]
+            self.cpp_info.defines= ["DEFINE_COOL=1"]
+
+The package information is done using the attributes of the :ref:`cpp_info_attributes_reference` object. This information will be aggregated
+by Conan and exposed via ``self.deps_cpp_info`` to consumers and generators.
+
+.. _package_information_components:
+
+Using Components
+----------------
+
+If your package contains more than one library or you want to define separated components so consumers can have more granular information,
+you can use components in your :ref:`method_package_info` method.
+
+.. warning::
+
+    This is a **experimental** feature subject to breaking changes in future releases.
+
+When you are creating a Conan package, it is recommended to have only one library (*.lib*, *.a*, *.so*, *.dll*...) per package. However,
+especially with third-party projects like Boost, Poco or OpenSSL, they would contain several libraries inside.
+
+Usually those libraries inside the same package depend on each other and modelling the relationship among them is required.
+
+With **components**, you can model libraries and executables inside the same package and how one depends on the other. Each library or
+executable will be one component inside ``cpp_info`` like this:
+
+.. code-block:: python
+
+    def package_info(self):
+        self.cpp_info.name = "OpenSSL"
+        self.cpp_info.components["crypto"].names["cmake_find_package"] = "Crypto"
+        self.cpp_info.components["crypto"].libs = ["libcrypto"]
+        self.cpp_info.components["crypto"].defines = ["DEFINE_CRYPTO=1"]
+        self.cpp_info.components["ssl"].names["cmake"] = "SSL"
+        self.cpp_info.components["ssl"].includedirs = ["include/headers_ssl"]
+        self.cpp_info.components["ssl"].libs = ["libssl"]
+        self.cpp_info.components["ssl"].requires = ["crypto"]
+
```

Comment:
I think at this moment it would be very good to highlight as "important" note, that yes, this is a bit of repeated information of what the package might have in its build scripts, but this is better:
- Not a big deal, typically define it once, not that terrible repetition, and it is nicely explicit and relatively intuitive
- The most important thing: This is generic and can be consumed by other build systems, different ones from the used one.

Reply:
I added the "important" box at the top as a general message. Please check it and I can move it down here if you want. thanks